### ISSUE TITLE
Gradle dependencyCheckAggregate task failed

### ISSUE THREAD

**Issue by SG-8** [dependency-check/DependencyCheck#7201]
Started getting this exception during gradle build in last 2-3 days only:

![image](https://github.com/user-attachments/assets/8ddb9bdb-cbb7-4ea6-bda0-98d2631a5a16)

Invocation of method getHighestSeverityText in class org.owasp.dependencycheck.dependency.Vulnerability threw exception java.lang.NullPointerException

**Comment by jeremylong** [comment]
What version of dependency-check are you using?

**Comment by aikebah** [comment]
@jeremylong my educated guess for this happening would be a vulnerability that has only CVSSv4 scoring attached to it (or one for which NVD has associated a CPE, but no CVSS score has been associated yet).

https://github.com/jeremylong/DependencyCheck/blob/main/core/src/main/java/org/owasp/dependencycheck/dependency/Vulnerability.java#L537-L545

**Comment by jeremylong** [comment]
@aikebah good catch - I suppose it is time I put together the PR to actually use the CVSS4v data in the reports... I hate velocity... ;)

**Comment by SG-8** [comment]
@jeremylong Thank you! In which version can we expect the fix?

**Comment by jeremylong** [comment]
12.0.0. I'll release 11.2.0 this week and do more testing before we release 12.0.0.

**Comment by aikebah** [comment]
Somehow overlooked in v12.0.0 release to be marked as solved
